management-server/cmd/adapter/main.go

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	logger "github.com/sirupsen/logrus"
```

Comment:
I think we can use CC logging package here https://github.com/wso2/product-microgateway/tree/main/adapter/pkg/logging

Reply:
If we use cc logger we need to add a package  config for that. Since its a main method we can just use this package as this right? 	https://github.com/wso2/product-microgateway/blob/9bdaac1269a78bedd2221de105507ab494ccf027/adapter/cmd/adapter/main.go#L48